flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/StandaloneAutoscalerEntrypoint.java

```diff
@@ -48,7 +49,9 @@ public class StandaloneAutoscalerEntrypoint {
 
     public static <KEY, Context extends JobAutoScalerContext<KEY>> void main(String[] args)
             throws Exception {
-        var conf = ParameterTool.fromArgs(args).getConfiguration();
+        var conf =
+                GlobalConfiguration.loadConfiguration(
+                        ParameterTool.fromArgs(args).getConfiguration());
```

Comment:
Would you mind extracting this part as a separate method and adding a unit test? 

Separate method is helpful for adding a unit test to check whether overwrite logic is fine.